src/coreclr/inc/llvm/ELF.h

```diff
@@ -1313,5 +1313,30 @@ enum {
   VER_NEED_CURRENT = 1
 };
 
+/* Note section contents.  Each entry in the note section begins with
+   a header of a fixed form.  */
+
+typedef struct
+{
+  Elf32_Word n_namesz;			/* Length of the note's name.  */
+  Elf32_Word n_descsz;			/* Length of the note's descriptor.  */
+  Elf32_Word n_type;			/* Type of the note.  */
+} Elf32_Nhdr;
+
+typedef struct
+{
+  Elf64_Word n_namesz;			/* Length of the note's name.  */
+  Elf64_Word n_descsz;			/* Length of the note's descriptor.  */
+  Elf64_Word n_type;			/* Type of the note.  */
+} Elf64_Nhdr;
+
+/* Known names of notes.  */
+
+/* Note entries for GNU systems have this name.  */
+#define ELF_NOTE_GNU		"GNU"
+
+/* Build ID bits as generated by ld --build-id.
+   The descriptor consists of any nonzero number of bytes.  */
+#define NT_GNU_BUILD_ID	3
```

Comment:
This file was forked from https://github.com/llvm/llvm-project/blob/main/llvm/include/llvm/BinaryFormat/ELF.h . It would be nice to keep out copy in sync with the upstream copy where possible.

For example, `NT_GNU_BUILD_ID` in the upstream copy looks like this:
```
// GNU note types.
enum {
  NT_GNU_ABI_TAG = 1,
  NT_GNU_HWCAP = 2,
  NT_GNU_BUILD_ID = 3,
  NT_GNU_GOLD_VERSION = 4,
  NT_GNU_PROPERTY_TYPE_0 = 5,
  FDO_PACKAGING_METADATA = 0xcafe1a7e,
};
```

Reply:
The latest llvm ELF.h is quite different from this one so I'm going to back my changes and save sync'ing it later.